src/net.cpp

```diff
@@ -686,7 +686,8 @@ const uint256& V1TransportDeserializer::GetMessageHash() const
     return data_hash;
 }
 
-CNetMessage V1TransportDeserializer::GetMessage(const CMessageHeader::MessageStartChars& message_start, int64_t time) {
+CNetMessage V1TransportDeserializer::GetMessage(const CMessageHeader::MessageStartChars& message_start, const std::chrono::microseconds& time)
```

Comment:
I always follow rule 1 (const&), because it is always cheap and prevents the overhead of having to think how many words a type has.

```
void f1(const string& s);  // OK: pass by reference to const; always cheap

Reply:
Changed, though.